modules/rds/main.tf

```diff
@@ -0,0 +1,39 @@
+module "rds" {
+  #checkov:skip=CKV_TF_1: Use specific version instead of defining the commit hash
+  source  = "terraform-aws-modules/rds-aurora/aws"
+  version = "7.6.0"
+
+  name = local.namespace
+
+  engine         = local.engine
+  engine_version = local.engine_version
+
+  availability_zones     = data.aws_availability_zones.available.names
+  vpc_id                 = var.vpc_id
+  subnets                = var.subnet_ids
+  vpc_security_group_ids = var.vpc_security_group_ids
+
+  instance_class = var.instance_type
+  instances = {
+    main = {}
+  }
+
+  autoscaling_enabled      = true
+  autoscaling_min_capacity = var.autoscaling_min_capacity
+  autoscaling_max_capacity = var.autoscaling_max_capacity
+
+  create_monitoring_role = var.create_monitoring_role
+  create_random_password = false
+  create_security_group  = false
+  storage_encrypted      = true
+
+  publicly_accessible = false
+
+  database_name       = var.database_name
+  master_username     = var.username
+  master_password     = var.password
+  port                = var.port
+  deletion_protection = false
```

Comment:
not sure why it should be `false`, isn't it safer to prevent deletion?

Reply:
Nice catch, due to testing purposes, I changed it to `false` and forgot to change it back. Thanks 🙏🏻 

7d3a356 ✅